main/components/ButtonGroup/stories/components/MultiSelect.tsx

```diff
@@ -0,0 +1,45 @@
+import {useTheme} from '@dooboo-ui/theme';
+import type {FC} from 'react';
+import {useState} from 'react';
+import {View} from 'react-native';
+import {ButtonGroup} from '../..';
+import {Button} from '../../../Button';
+import {Typography} from '../../../Typography';
+
+const MultiSelect: FC = () => {
+  const data = ['Item 1', 'Item 2', 'Item 3', 'Item 4'];
+
+  const {theme} = useTheme();
+
+  const [selectedIndices, setSelectedIndices] = useState([0]);
+
+  return (
+    <View>
+      <Typography.Heading1 style={{paddingVertical: 20}}>
+        Multi Select
+      </Typography.Heading1>
+      <ButtonGroup
+        data={data}
+        selectedIndices={selectedIndices}
+        borderStyle={{color: theme.button.secondary.bg, width: 1}}
+        renderItem={({item, selected, index: itemIndex}) => (
+          <Button
+            text={item}
+            rounded={0}
+            type={selected ? 'solid' : 'text'}
+            color={'secondary'}
+            onPress={() =>
+              setSelectedIndices((prev) =>
+                prev.includes(itemIndex)
+                  ? prev.filter((v) => v !== itemIndex)
+                  : [...prev, itemIndex],
+              )
+            }
+          />
+        )}
+      />
```

Comment:
I think `multi-select` is over spec for [ButtonGroup] 🤔
[ButtonGroup] is used for switching nested screen and we don't need to show two screens at a time.

Reply:
Before starting ButtonGroup renewal, I looked up for other popular React Native libraries and how they do. 
For Example, Button Group in [React Native Element -ButtonGroup](https://reactnativeelements.com/docs/components/buttongroup) also has API for multiple selection. 

However, I also agree with you that `multi-select` might be over spec. 
Let me know how you want to do with `multi-select` spec.